tests/integration/tests/bundles/overlays/local-charm-overlay.yaml.j2

```diff
@@ -1,5 +1,7 @@
 applications:
   {{ charm_name }}:
     charm: {{ CHARM_LOCATION }}
+    {% if CHARM_SNAP_LOCATION %}
     resources:
       openstack-exporter: {{ CHARM_SNAP_LOCATION }}
+    {% endif %}
```

Comment:
Please document this in CONTRIBUTING.md (that it will use a local resource if there is a particularly named snap file in the root of the repo, that it relies on env vars set from the makefile).